package.json

```diff
@@ -169,6 +169,7 @@
     "jest-circus": "27.0.6",
     "jest-fetch-mock": "^3.0.3",
     "jest-junit": "^9.0.0",
+    "jest-sentry-environment": "1.1.4",
```

Comment:
I wonder if we can write this in typescript now that it's in a npm module that can have a build 👀

Reply:
Not that we couldn't before I guess 😆

Reply:
We could, I'm not going to do that - I'd prefer (if this library does gain traction), that the SDK team takes it into their monorepo